modules/ossm-threescale-istio-adapter-troubleshooting.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v1x/threescale_adapter/threescale-adapter.adoc
+// * service_mesh/v2x/threescale_adapter/threescale-adapter.adoc
+
+[id="ossm-threescale-istio-adapter-verification-troubleshooting_{context}"]
+= 3scale Istio adapter verification and troubleshooting
+
+You might want to check whether the 3scale Istio adapter is working as expected. If your adapter is not working, use the following steps to help troubleshoot the problem.
+
+.Procedure
+
+. Ensure the _3scale-adapter_ pod is running in the control plane namespace:
++
+----
+$ oc get pods -n <istio-system>
+----
+. Check that the _3scale-adapter_ pod has printed out information about itself booting up, such as its version:
++
+----
+$ oc logs <istio-system>
+----
+. When performing requests to the services protected by the 3scale adapter integration, always try requests that lack the right credentials and ensure they fail. Check the 3scale adapter logs to gather additional information. See link:https://docs.openshift.com/container-platform/4.7/support/troubleshooting/investigating-pod-issues.html#inspecting-pod-and-container-logs_investigating-pod-issues[Inspecting pod and container logs].
```

Comment:
Xrefs inside modules are still banned in OpenShift docs.  As a workaround, you could split this topic into two modules and put the xref in the assembly between the includes for the two modules.